atef/widgets/config/utils.py

```diff
@@ -1264,6 +1321,10 @@ def child(self, row: int) -> TreeItem:
         if row >= 0 and row < self.childCount():
             return self._children[row]
 
+    def get_children(self) -> Generator[TreeItem, None, None]:
+        """ Yield this item's children """
```

Comment:
This one may make you yell at your monitor (sorry!) - style guide does say "no extra whitespace with a single line docstring" [ref](https://confluence.slac.stanford.edu/display/PCDS/ECS+Python+Standards+and+Style+Guide#ECSPythonStandardsandStyleGuide-SinglelineDocstrings)

It's another thing we should be linting for at some point

Reply:
I'll fix this for the entire repo, at the expense of the diff